packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3886,13 +3886,16 @@ export default class Meeting extends StatelessWebexPlugin {
       return Promise.reject(error);
     }
 
-    return this.brbState.enable(enabled, this.sendSlotManager).then(() => {
-      if (this.audio && enabled) {
-        // locus mutes the participant with brb enabled request,
-        // so we need to explicitly update remote mute for correct logic flow
-        this.audio.handleServerRemoteMuteUpdate(this, enabled);
-      }
-    });
+    return this.brbState
+      .enable(enabled, this.sendSlotManager)
+      .then(() => {
+        if (this.audio && enabled) {
+          // locus mutes the participant with brb enabled request,
+          // so we need to explicitly update remote mute for correct logic flow
+          this.audio.handleServerRemoteMuteUpdate(this, enabled);
+        }
+      })
+      .catch((error) => Promise.reject(error));
```

Comment:
```suggestion
		await this.brbState.enable(enabled, this.sendSlotManager);
		
		if (this.audio && enabled) {
        // locus mutes the participant with brb enabled request,
        // so we need to explicitly update remote mute for correct logic flow
        this.audio.handleServerRemoteMuteUpdat
		}
```

With this we don't need to re-throw the error and we're using that we are in `async` method

Reply:
In code style we pass Promise.reject(error); without await ( because we need this error outside of this function )